template/python/Dockerfile

```diff
@@ -1,27 +1,34 @@
 FROM python:2.7-alpine
 
+RUN adduser app -D
```

Comment:
Templates need to add a group too so they follow the pattern of the Node example.

```
RUN addgroup -S app && adduser -S -g app app
```

Then `chown to app:app`

Please also `chmod /tmp to 777` for all templates to get around buildkit/containerd issue.